devices/libsdio/README.md

```diff
@@ -0,0 +1,277 @@
+# libsdio
+
+## Description
+
+libsdio is a static, precompiled library containing a generic SDIO driver which directly controls platform hardware. This driver defines an API providing basic interface control functionality.
+
+## Platform support
+
+| Target            | Supported?       |
+|-------------------|------------------|
+| armv7a7-imx6ull   |:heavy_check_mark:|
+| armv7m7-imxrt106x |:x:               |
+| ia32-generic-pc   |:x:               |
+
+## Limitations
+
+- only one library instance per system
+- no fallback SD protocol support
+- no support for multiple SDIO devices
+- no support for SDIO in SPI mode
+- only 4-bit data transfer mode supported
+- only 50/25 MHz clock speeds available
```

Comment:
Just imx6ull version limitation, not whole lib in general

Reply:
Appending `on imx6ull` to the statement in question.